Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```
@@ -2,39 +2,63 @@
 
 .bocCheckBox
 {
-  margin-right: 0.5em;
+  display: inline-flex;
+  align-items: center;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox span.description
+{
+  display: inline-flex;
+  align-items: center;
+}
+
+.bocCheckBox img,
+.bocCheckBox input ~ span.image-visualizer::before
+{
+  margin: var(--remotion-themed-padding);
   vertical-align: middle;
-  display: inline-block;
-  white-space: nowrap;
+  height: 1.5em;
+  width: 1.5em;
 }
 
-.bocCheckBox.readOnly
+.bocCheckBox input ~ span.image-visualizer::before
 {
+  visibility: visible;
+  background-image: url('../Image/sprite.svg#CheckBoxFalse');
+  display: inline-block;
+  content: "";
+  z-index: 1;
+  position: relative;
 }
 
-.bocCheckBox.disabled
+.bocCheckBox input[checked] ~ span.image-visualizer::before
 {
+  background-image: url('../Image/sprite.svg#CheckBoxTrue');
 }
 
-.bocCheckBox input
+/* Fixes for element focus */
+
+.bocCheckBox span[role=checkbox]:focus
 {
-  margin-left: 4px;
-  margin-right: 3px;
-  padding: 0px;
-  vertical-align: -0.1em;
+  outline: none;
 }
 
-.bocCheckBox span + span
+.blink .bocCheckBox span[role=checkbox]:focus > img,
+.blink .bocCheckBox input:focus + span::before
```

should this be 
```suggestion
.blink .bocCheckBox input:focus + span.image-visualizer::before
```
?